pyuvsim/simsetup.py

```diff
@@ -595,20 +662,21 @@ def parse_frequency_params(freq_params):
     Parse the "freq" section of obsparam.
 
     Args:
-        freq_params: Dictionary of frequency parameters
+        freq_params: Dictionary of frequency parameters.
+            See pyuvsim documentation for examples of allowable key combinations.
```

Comment:
more explicit directions about where to find the info would be better that "pyuvsim documentation".